docs/markdown/Using Pants/troubleshooting.md

```diff
@@ -233,8 +233,7 @@ This can be fixed by setting `ulimit -n 10000`. (10,000 should work in all cases
 > We recommend permanently setting this by either:
 > 
 > 1. Adding `ulimit -n 10000` to your `./pants` script.
```

Comment:
Oh, while being here, I think we might as well update the recommendation to patch the `./pants` script to instead refer to the `.pants.bootstrap` script as a means to patch the pants runtime env, if you don't mind :) 

https://github.com/pantsbuild/setup/blob/ab8581bb52d92fcb3fda1b1dfb20204ae8888a90/pants#L19-L23

Perfectly fine if not, I wasn't aware of this patching pants script being a thing in the docs.

Reply:
```suggestion
> 1. Adding `ulimit -n 10000` to your `./pants.bootstrap` script.
```

Reply:
Thanks, minor nit, there's a missing leading `.` on the file name :)

Reply:
@Eric-Arellano oops, I missed this convo was marked as resolved..

Reply:
Oh, this is supposed to be `./pants` bootstrap script?

Reply:
No, it was, I suggested this change, but he missed the leading dot, so `./.pants.bootstrap`.

Reply:
Or just `.pants.bootstrap`..

Reply:
@michaelglass no worries, it was a double-miss between me and Eric :)